platform/ios/NSBundle+MGLAdditions.m

```diff
@@ -4,6 +4,8 @@
 
 @implementation NSBundle (MGLAdditions)
 
+void linkBundleCategory(){}
```

Comment:
These functions should have an `mgl_` prefix to avoid potential conflicts with client code that uses the same approach.